samples/base/main.cpp

```diff
@@ -1,13 +1,21 @@
 #include <lug/Config.hpp>
 #include <lug/System/Exception.hpp>
+#include <lug/System/Logger.hpp>
 #include <iostream>
 
-int main() {
-    try {
+int main(int, const char*[]) {
```

Comment:
Remove all the useless comments and remove the `const` (It can be a problem in the future)